Samples/Python/add_node.py

```diff
@@ -0,0 +1,90 @@
+import re
+import requests
+import swisclient
+
+
+def main():
+    npm_server = 'localhost'
+    username = 'admin'
+    password = ''
+
+    swis = swisclient.SwisClient(npm_server, username, password)
+    print("Add an SNMP v2c node:")
+  
+    # fill these in for the node you want to add!
+    node_name = ''
+    ip_address = ''
+    community = ''
+
+    # set up property bag for the new node
+    props = {
+        'Caption': node_name,
+        'IPAddress': ip_address,
+        'DynamicIP': False,
+        'EngineID': 1,
+        'Status': 1,
+        'Allow64BitCounters': 1,
+        'ObjectSubType': 'SNMP',
+        'SNMPVersion': 2,
+        'SysObjectID': '',
```

Comment:
Many of these properties are just being explicitly set to their default values. Please see https://github.com/solarwinds/OrionSDK/blob/master/Samples/PowerShell/CRUD.AddNode.ps1 for the set of properties you can omit or comment out.